<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #D9D9D9;
	color: black;
	border: 1.5px solid black;
	border-right: none;
	border-bottom: none;
	padding: 10px 20px;
	font-size: 24px;
	font-weight: bold;
}
QPushButton:hover {
	background-color: #e0e0e0;
}
QPushButton:checked { /* 선택된 탭처럼 보이게 하려면 */
	background-color: #08F28C; /* 민트색 */
}

QPushButton#BtnMyInfo { /* 마지막 버튼의 objectName이 'lastButton'이라고 가정 */
    border-right: 1.5px solid black;
}

QPushButton#btnMyInfo {
    border-right: 1.5px solid black;
}</string>
   </property>
   <widget class="QPushButton" name="btnMyInfo">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>0</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans KR</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>내 정보</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans KR</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>설정</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMain">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans KR</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>시간표</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/side_logo.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1600</width>
      <height>870</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
	border-top: 1.5px solid black;
}</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <widget class="QWidget" name="timetableWidget">
     <attribute name="title">
      <string>1</string>
     </attribute>
     <widget class="QTableWidget" name="timetable">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>1200</width>
        <height>780</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
	border: 3px solid black;
}

QHeaderView::section {
	border: 1px solid black;
	font-size: 36px;
	padding: 5px;
	background-color: white;
	font-weight: bold;
 	text-align: center;
}

QTableWidget::item {
	font-size: 24px;
	border: 1px solid black;
}

QTableWidget::item:selected{
	background-color: lightgray;
	color: black;

}</string>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <row>
       <property name="text">
        <string>1교시</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2교시</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3교시</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4교시</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5교시</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6교시</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7교시</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>월</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>화</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>금</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>1600</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans KR</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>이번주 시간표</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingWidget">
     <attribute name="title">
      <string>2</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>셋팅</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="myinfoWidget">
     <attribute name="title">
      <string>3</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>마이인포</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
